checker.py

```diff
@@ -411,7 +164,7 @@ def is_interesting_with_empty_marker_bodies(self, case: utils.Case) -> bool:
             with open(tf.name, "w") as f:
                 f.write(empty_body_code)
```

Comment:
Something like this is needed in the reduction check?

Reply:
:+1:
Basically we need an arbitrary preprocessing function for each part of the sanitizer. More higher-order functions :D